tools/i18n/locales.yaml

```diff
@@ -0,0 +1,59 @@
+zh_CN:
+    标题: '快速、高质量的零样本文本转语音，支持 “流匹配”'
+    音频生成: ''
+    请输入目标文本: ''
+    请上传参考音频（15秒以内）: ''
+    请上传参考文本: ''
+    预测音频文本: ''
+    自动预测文本: ''
+    请输入生成文本: ''
+    🔵语速调节：: ''
+    🔵推理步数：: ''
+    🎉 推理语音: ''
+    推理结果: ''
+    输出消息: ''
+
+en_US:
+    标题: 'Fast and High-Quality Zero-Shot Text-to-Speech with Flow Matching'
+    音频生成: 'Audio generation'
+    请输入目标文本: 'Please enter the target text'
+    请上传参考音频（15秒以内）: 'Please upload a reference audio (within 15s)'
+    请上传参考文本: 'Please upload reference text'
+    预测音频文本: 'Predictive audio text'
+    自动预测文本: 'Automatic predictive text'
+    请输入生成文本: 'Please enter the generated text'
+    🔵语速调节：: '🔵 Audio speed'
+    🔵推理步数：: '🔵 Number of steps'
+    🎉 推理语音: '🎉 Inferential speech'
+    推理结果: 'Inference results'
+    输出消息: 'Output message'
+
+ja_JP:
+    标题: 'フローマッチングによる高速で高品質なゼロショットテキスト読み上げ'
+    音频生成: 'オーディオ生成'
+    请输入目标文本: '対象のテキストを入力してください'
+    请上传参考音频（15秒以内）: '参考音声をアップロードしてください(15秒未満)'
+    请上传参考文本: '参照テキストをアップロードしてください'
+    预测音频文本: '予測音声テキスト'
+    自动预测文本: '自動予測テキスト'
+    请输入生成文本: '生成されたテキストを入力してください'
+    🔵语速调节：: '🔵 オーディオ速度'
+    🔵推理步数：: '🔵 推論の手順'
+    🎉 推理语音: '🎉 推論音声'
+    推理结果: '推論結果'
+    输出消息: '出力メッセージ'
+
+ko_KR:
```

Comment:
I think supporting Chinese and English is enough. We can't make sure the translation of other languages are accurate.